src/steps/os/unix.rs

```diff
@@ -749,6 +749,16 @@ pub fn run_maza(ctx: &ExecutionContext) -> Result<()> {
     ctx.run_type().execute(maza).arg("update").status_checked()
 }
 
+pub fn run_aqua(ctx: &ExecutionContext) -> Result<()> {
```

Comment:
Have you tried the dry run option, does it work with this new step?

```sh
$ topgrade --only aqua --dry-run
```

Also the `--yes` option, does Aqua have this option or something similar?

>  Optional: The --dry-run option works with this step
 Optional: The --yes option works with this step if it is supported by
the underlying command

Reply:
There isn't any -y or --yes on the update commands, So i don't think there is a dry run available.

Reply:
> There isn't any -y or --yes on the update commands

Get it, then we don't need to worry about this.

Dry run is another option, which is not related to --yes, when it is passed to Topgrade, Topgrade should simply echo what will be done and exit, does this PR implement it?

Reply:
There isn't a dry-run option either.

Reply:
So 

```sh
$ topgrade --only aqua --dry-run
```

would actually do the update, right? We need to emulate this `--dry-run` option on the Topgrade's side, see:

https://github.com/topgrade-rs/topgrade/blob/a2375b48203322e39d36a6235db6aabdc4d3e5e0/src/self_update.rs#L20-L23

for an example.